Floofbot/Modules/Tag.cs

```diff
@@ -7,191 +7,188 @@
+        [Command("remove")]
+        [RequireUserPermission(GuildPermission.ManageMessages)]
+        public async Task Remove(string tag)
+        {
+            string tagId = $"{tag.ToString()}:{Context.Guild.Id.ToString()}";
 
-                string select = @"SELECT COUNT(*) FROM Tags WHERE TagID = $TagId";
-                SqliteCommand command = new SqliteCommand(select, dbConnection);
-                command.Parameters.Add(new SqliteParameter("$TagId", tagId));
+            string select = @"SELECT COUNT(*) FROM Tags WHERE TagID = $TagId";
+            SqliteCommand command = new SqliteCommand(select, dbConnection);
+            command.Parameters.Add(new SqliteParameter("$TagId", tagId));
 
-                if (Convert.ToInt32(command.ExecuteScalar()) > 0) {
-                    string delete = @"DELETE FROM Tags WHERE TagID = $TagId";
-                    command = new SqliteCommand(delete, dbConnection);
-                    command.Parameters.Add(new SqliteParameter("$TagId", tagId));
-                    command.ExecuteScalar();
+            if (Convert.ToInt32(command.ExecuteScalar()) > 0) {
+                string delete = @"DELETE FROM Tags WHERE TagID = $TagId";
+                command = new SqliteCommand(delete, dbConnection);
+                command.Parameters.Add(new SqliteParameter("$TagId", tagId));
+                command.ExecuteScalar();
 
-                    await sendEmbed(createDescriptionEmbed($"💾 Tag: `{tag}` Removed"));
-                }
-                else {
-                    await sendEmbed(createDescriptionEmbed($"💾 Could not find Tag: `{tag}`"));
-                }
+                await SendEmbed(CreateDescriptionEmbed($"💾 Tag: `{tag}` Removed"));
             }
-
-            [Command("remove")]
-            [Priority(1)]
-            public async Task Remove()
-            {
-                await sendEmbed(createDescriptionEmbed($"💾 Usage: `tag remove [name]`"));
+            else {
+                await SendEmbed(CreateDescriptionEmbed($"💾 Could not find Tag: `{tag}`"));
             }
+        }
 
-            [Command]
-            [RequireUserPermission(GuildPermission.AttachFiles)]
-            public async Task GetTag(string Tag = "")
-            {
-                if (!string.IsNullOrWhiteSpace(Tag)) {
-                    string TagID = $"{Tag}:{Context.Guild.Id}".ToLower();
-                    string sql = @"SELECT Content FROM Tags
-                                 Where TagID = $TagID";
-                    SqliteCommand command = new SqliteCommand(sql, dbConnection);
-                    command.Parameters.Add(new SqliteParameter("$TagID", TagID));
-                    var result = command.ExecuteScalar();
+        [Command("remove")]
+        [Priority(1)]
+        public async Task Remove()
+        {
+            await SendEmbed(CreateDescriptionEmbed($"💾 Usage: `tag remove [name]`"));
+        }
+
+        [Command]
```

Comment:
No command name?

Reply:
That's correct. This case handles commands like ".tag ..."

Reply:
Oh I see now!